### PR TITLE
CRM-19741 Add back in fix and add in unit tests to lock in fix

### PR DESCRIPTION
**Pull request by seamuslee001**
* [CRM-19741: Price set and price set value "label" fields are inconsistant to users](https://issues.civicrm.org/jira/browse/CRM-19741)

### CODE REVIEW

**Comment by seamuslee001** [comment]
This fixes regression now @jitendrapurohit @eileenmcnaughton and adds unit tests as well

**Comment by eileenmcnaughton** [comment]
@jitendrapurohit can you check this - it is the regression you spotted

File: CRM/Price/BAO/PriceField.php [hunk 1]
```diff
@@ -170,7 +176,7 @@ public static function create(&$params) {
           throw new CRM_Core_Exception($e->getMessage());
         }
       }
-      elseif (!empty($optionIds)) {
+      elseif (!empty($optionsIds)) {
         $optionsLoad = civicrm_api3('price_field_value', 'get', array('id' => $optionIds['id']));
```
Comment (jitendrapurohit) [inline CRM/Price/BAO/PriceField.php]: seems an additional type here - `$optionIds` - failing the test again?

Comment (seamuslee001) [inline CRM/Price/BAO/PriceField.php]: Thanks for the ping have fixed it up (minor spelling issues mostly)

**Comment by eileenmcnaughton** [comment]
@jitendrapurohit is this good to merge now?

**Review by jitendrapurohit — APPROVED** [review]
@eileenmcnaughton yeah, confirmed that it works well. Thanks for the test @seamuslee001

**Pull request merged**